source/release_notes/5.0.13-11.rst

```diff
@@ -0,0 +1,60 @@
+.. _PSMDB-5.0.13-11:
+
+================================================================================
+*Percona Server for MongoDB* 5.0.13-11 (2022-10-)
+================================================================================
+
+:Date: October , 2022
+:Installation: `Installing Percona Server for MongoDB <https://www.percona.com/doc/percona-server-for-mongodb/5.0/install/index.html>`_
+
+Percona Server for MongoDB 5.0.13-12 is an enhanced, source-available, and highly-scalable database that is a
+fully-compatible, drop-in replacement for `MongoDB 5.0.13 <https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.12---sep-05--2022>`_ and `MongoDB 5.0.13 Community Edition <https://www.mongodb.com/docs/manual/release-notes/5.0/#5.0.13---sep-29--2022>`_.
+
+It supports protocols and drivers of both MongoDB 5.0.12 and 5.0.13.
+
+This release of |PSMDB| includes the improvements and bug fixes of MongoDB  Community Edition 5.0.12 and 5.0.13. 
+
+Release Highlights
+==================
+
+* :ref:`Data at rest encryption using the KMIP protocol <kmip>` is now generally available, enabling you to use this functionality in production environments.
+* |PSMDB| Docker container now includes the ``mongodb-tools`` utility set to align with the upstream container. This enables users to manage backup/restore operations of |PSMDB|.
```

Comment:
Can we possibly add some link here to the documentation on how to use mongodb-tools?